shopwild-frontend/src/components/ProductDetails/ProductDetails.jsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
+
+import "./ProductDetails.css";
+
+import { useData } from "../../context/data-context";
+
+import Toast from "../Toast/Toast";
+
+const ProductDetails = () => {
+  const [showToast, setShowToast] = useState(false);
+  const [toastMessage, setToastMessage] = useState(null);
+
+  const { state: productId } = useLocation();
+  const navigate = useNavigate();
+
+  const { state, dispatch } = useData();
+
```

Comment:
Try using a custom useDocumentTitle hook whenever a user is moving from one page to other 
change the title according to it